connector/src/main/java/org/geysermc/connector/network/translators/chat/MessageTranslator.java

```diff
@@ -114,8 +122,8 @@ public static String convertMessageLenient(String message, String locale) {
             String convertedMessage = convertMessage(convertToJavaMessage(message), locale);
```

Comment:
It won't let me go any higher - `isMessage` is only called here. Perhaps we should just do the `try/catch` here so we aren't deserializing a message twice?

Reply:
Agreed.